lesson02/elb_asg/common_vars.tf

```diff
@@ -0,0 +1,36 @@
+variable "aws-region" {
```

Comment:
shall we use `snake_case` for all our variables and resources names to be consistent?

Reply:
same for: elb-listener, elb-health-check